src/components/layout/GenericLayout/Footer/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import styled from 'styled-components'
+import { depositApi } from 'api'
+
+// Components
+import { BlockExplorerLink } from 'components/common/BlockExplorerLink'
+
+// Hooks
+import { useWalletConnection } from 'hooks/useWalletConnection'
+
+// Config
+import { footerConfig } from '../Footer/config'
+
+const FooterStyled = styled.footer`
+  position: fixed;
+  font-size: 1.1rem;
+  bottom: 0;
+  height: 3rem;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0 1rem;
+  border-top: 0.1rem solid var(--color-border);
+  flex: 1 1 auto;
+  background: var(--color-gradient-1);
+  width: 100%;
+  color: var(--color-text-secondary2);
+  justify-content: space-between;
+`
+
+const VerifiedButton = styled(BlockExplorerLink)`
+  margin: 0 auto 0 0.6rem;
+`
+
+const VersionsWrapper = styled.div`
+  display: flex;
+  margin: 0 0 0 auto;
+
+  > a {
+    text-decoration: none;
+
+    &:focus,
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  > a:not(:last-of-type) {
+    margin: 0 1.6rem 0 0;
+    flex-flow: row nowrap;
+    display: flex;
+    position: relative;
+
+    &::after {
+      content: '-';
+      position: absolute;
+      right: -1rem;
+      display: block;
+    }
+  }
```

Comment:
```suggestion
  > a:not(:last-of-type)::after {
    content: "  -  ";
    white-space: pre;
}
```
I feel this would accomplish about the same with less css

Reply:
Yes, except that on hover it also applies the underline to the `-` character:
<img width="102" alt="Screen Shot 2020-11-23 at 09 52 20" src="https://user-images.githubusercontent.com/31534717/99943754-15c9cb00-2d72-11eb-80c6-a28c1a7b6057.png">

Tried doing text-decoration: none; on the parent `a` element, and then play with border-bottom. But same result otherwise. Using position: absolute in this case mitigates it...

Reply:
Understandable, have a nice day